api-examples/wordpress-plugin/sugarcrm-api/js/scripts.js

```diff
@@ -0,0 +1,51 @@
+
+function callAuth(){
+
+data = {
+    	username: jQuery('#sugarcrm_input_username').val(),
+    	password: jQuery('#sugarcrm_input_password').val(),
+    	url: jQuery('#sugarcrm_input_url').val(),
+    	version: jQuery('#sugarcrm_input_api_version option:checked').val(),
+    }
+jQuery.ajax({
+    url: "http://localhost:8888/sugartest/wp-json/sugarcrm-api/v1/auth",
```

Comment:
make this either relative or grab the base url from window.location to build the URL